backend/users/models.py

```diff
@@ -0,0 +1,35 @@
+from django.contrib.auth.models import (
+    AbstractBaseUser,
+    BaseUserManager,
+    PermissionsMixin,
+)
+from django.db import models
+
+
+class CustomUserManager(BaseUserManager):
+    def create_user(self, email, password=None, **extra_fields):
+        if not email:
+            raise ValueError("Email field must be set")
+        email = self.normalize_email(email)
+        user = self.model(email=email, **extra_fields)
+        user.set_password(password)
+        user.save(using=self._db)
+        return user
```

Comment:
Django provides built-in validation for model fields which can be leveraged to avoid hardcoding error messages. 
When you attempt to save a model instance, Django performs a full_clean() on the model which includes a call to the model's validate_unique() method, ensuring field uniqueness where required.

by chatgpt 😉

Reply:
喔喔喔感謝～原來有這個 XD 不過這個 model 只是為了測試 graphql 是否成功運行，之後應該也會整個砍掉就是了。
我再改掉這個，讓之後開發的也能知道有這個，thanks～